docs/src/main/java/io/quarkus/docs/generation/CopyJavaExamples.java

```diff
@@ -0,0 +1,63 @@
+package io.quarkus.docs.generation;
```

Comment:
i'm all for using java for scripting but any reason this couldn't be done using maven resource plugin?

my guess is that it might be limited to what is actually "inside" the project ....this does mean the docs project is now dependent on resources outside itself and thus cannot easily be copied/moved. Might not be a big deal but just feels so dirty :/

Reply:
This is more like what we do for config.. which is why I did it this way.
Config is also a 'gather a bunch of stuff from other projects' effort.